lib/connect/api.js

```diff
@@ -3,6 +3,28 @@ const cli = require('heroku-cli-util')
 const co = require('co')
 const https = require('https')
 const querystring = require('querystring')
+const axios = require('axios')
+
+function makeConnectClient (context) {
+  return axios.create({
+    headers: {
+      'Content-Type': 'application/json',
+      'Authorization': `Bearer ${context.auth.password}`,
+      'Heroku-Client': 'cli'
+    }
+  })
+}
+
+function makeDiscoveryClient (context) {
+  return axios.create({
+    baseURL: 'https://hc-central-qa.herokai.com',
```

Comment:
TODO: This shouldn't point at QA

Reply:
We should have an env var that lets us override this and default to prod

Reply:
@lambacck Interesting. So this would be totally environmental, not a flag that user could pass?

Reply:
@mattrothenberg the CLI allows you to override with `HEROKU_HOST` so I think it would be best if we did something similar.

Reply:
@sigmavirus24 Not sure I follow, can you link to a commit or documentation that illustrates this? 

This is the technique we're going to use for switching allowing user to target QA: https://github.com/heroku/heroku-connect-plugin/pull/99/commits/694f85498e8f1e4a080500698c36e03f71cb41b6